draft-enghardt-panrg-path-properties.md

```diff
@@ -66,63 +64,62 @@ in Path-Aware Networking {{I-D.irtf-panrg-questions}}, which is a product of the
 
 Monetary Cost:
-: This is information related to billing, data caps, etc. It could be the allowed monthly data cap, the start and end of a billing period, the monetary cost per Megabyte sent or received, etc.
+: The price to be paid to transmit a specific flow across a set of path elements.
+
+Presence of a certain network function on the path:
+: Indicates that a certain path element performs a certain network function on a flow, e.g., whether the path element acts as a proxy, as a firewall, or performs Network Address Translation (NAT). This path element may be either in the same administrative domain as the endpoint or in a different administrative domain, i.e., the backbone.
 
 
 # Backbone Properties
 
-Backbone path properties relate to non-dynamic path properties that are not within the endpoint's domain.
-They are likely to stay constant within the lifetime of a connection, since Internet "backbone" routes change infrequently.
-These properties usually change on the timescale of seconds, minutes, or hours, when the route changes.
+Backbone path properties relate to path elements that are in a different administrative domain than an endpoint considering them, thus, in the backbone from the endpoint's point of view.
 
-Even if these properties change, endpoints can neither specify which backbone nodes to use, nor verify data was sent over these nodes.
-An endpoint can for example choose its access provider, but cannot choose the backbone path to a given destination since the access provider will make their own policy-based routing decision.
+Typically, backbone properties are less easily available to an endpoint than domain properties, due to the potential increased distance and the lack of pre-existing trust or contractual relationship.
 
-Presence of certain device on the path:
-: Could be the presence of a certain kind of middlebox, e.g., a proxy, a firewall, a NAT.
+Additionally, endpoints are less likely to be able to influence which path elements form their path in the backbone, as well as their properties.
 
-Presence of a packet forwarding node or specific Autonomous System on a path:
-: Indicates that traffic goes through a certain node or AS, which might be relevant for deciding the level of trust this path provides.
+Some path properties relate to the entire path, part of which often lies outside of an endpoint's administrative domain. Thus, such properties are listed as Backbone Properties.
+
+
+Administrative Entity:
+: The administrative entity, e.g., the AS, to which a path element or set of path elements belongs.
 
 Disjointness:
-: How disjoint a path is from another path.
+: For a set of two paths, the set of path elements which are shared.
```

Comment:
I would extend the description of disjointness a bit:

"For a set of two paths, the number of shared path elements can be a measure of intersection (e.g., Jaccard coefficient, which is the number of shared elements divided by the total number of elements), and conversely, the number of non-shared path elements can be a measure of disjointness (e.g., 1 - Jaccard coefficient).
A multipath protocol might use disjointness of paths as a metric to reduce the number of single points of failure."

Reply:
Nice. I didn't know about this and it seems pretty useful. Thanks!